gateway-ha/src/test/java/io/trino/gateway/ha/router/TestRoutingGroupSelector.java

```diff
@@ -85,11 +85,11 @@ void testByRoutingGroupHeader()
         // If the header is present the routing group is the value of that header.
         when(mockRequest.getHeader(ROUTING_GROUP_HEADER)).thenReturn("batch_backend");
         assertThat(RoutingGroupSelector.byRoutingGroupHeader().findRoutingGroup(mockRequest))
-                .isEqualTo("batch_backend");
+                .contains("batch_backend");
```

Comment:
why do we need to change this?

Reply:
changing this because `findRoutingGroup` returns an `Optional<String>` now